### ISSUE TITLE
Redact username & email from logs output

### ISSUE THREAD

**Issue by mrpher** [balena-io/deploy-to-balena-action#117]
For public repos, some users might not want their Balena username to be visible to everyone in the Actions logs.
Would be nice if the output could be partially redacted. For example:

```
== ACCOUNT INFORMATION
USERNAME: us*******e123
EMAIL:    us*******ail.com
URL:      balena-cloud.com
git version 2.24.4
```

**Comment by mrpher** [comment]
On second thought, some might want URL redacted as well, especially for open-balena instances where a user may not want that publicly exposed.

**Comment by 20k-ultra** [comment]
I totally agree with redacting the PII here! In fact, I feel bad now for not thinking about this...I will modify the entrypoint to not even log these at all (don't bother redacting).

For the domain I don't really believe in security by obscurity so hiding the domain is not as critical...it is useful to confirm you're pointed at the right environment so I think we should leave it.

**Comment by 20k-ultra** [comment]
Resolved with https://github.com/balena-io/deploy-to-balena-action/pull/120.

I noted in the PR description that redacting the domain also isn't required since if you can see logs then you have access to the repo which means you could also see what domain is passed to the environment input. I am ok to reconsider though if more people really feel it's a risk for them however I personally do not see any value in hiding a domain in hopes of protecting it.

**Comment by 20k-ultra** [comment]
fixed in v0.5.12
